src/components/flight-views/flight-edit-view.jsx

```diff
@@ -413,9 +413,12 @@ export default class FlightEditView extends React.Component {
             <SectionRow>
               <DateInput
                 inputDateValue={this.state.item.date}
-                inputTimeValue={this.state.item.time || ''}
```

Comment:
no need for this, input of type `time` can handle `null` and `undefined` values just fine

Reply:
actually when this is removed, there are errors on the console when adding a new flight:
`react-dom.development.js?61bb:506 Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
in input (created by DateInput) ` / `Warning: A component is changing an uncontrolled input of type time to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
in input (created by DateInput)
in div (created by InputContainer) `

Reply:
You are absolutely right! My bad, forgot about it.

It doesn't complain for existing flights though 🤔  Oh, I see. `FlightModel.getEditOutput` returns either a value or an empty string. I think for consistency I will change `FlightModel.getNewItemOutput` to return an empty string as well. And will change configuration for time field to have `defaultVal: null`, so we don't save empty strings to the Database. It makes things easier, if you want to get all the flights with empty time you don't need to query it like `WHERE time = NULL OR time = ''`.

Committed directly to `flight_sorting` branch (2 commits): [861a2fb](https://github.com/kaplona/koifly/pull/30/commits/861a2fbeaa212f07d4094f1e6ceb9c8be37e9d79) and [3d0f04e](https://github.com/kaplona/koifly/pull/30/commits/3d0f04ea52b3508e5c0397950d7d49cea22a9e89)